tools/StaticAnalysis/Exceptions/Az.Websites/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,2 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Az.Websites","Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps.EditAzureWebAppBackupConfiguration","Edit-AzWebAppBackupConfiguration","0","3030","The generic type for 'property UserAssignedIdentities' has been changed from 'System.Collections.Generic.IDictionary`2[System.String,Microsoft.Azure.Management.WebSites.Models.ManagedServiceIdentityUserAssignedIdentitiesValue]' to 'System.Collections.Generic.IDictionary`2[System.String,Microsoft.Azure.Management.WebSites.Models.UserAssignedIdentity]'. ","Change the generic type for 'property UserAssignedIdentities' back to 'System.Collections.Generic.IDictionary`2[System.String,Microsoft.Azure.Management.WebSites.Models.ManagedServiceIdentityUserAssignedIdentitiesValue]'."
```

Comment:
Instead of suppressing every breaking change, we should consider how it's gonna affect our customers; is there a way to mitigate?

Reply:
Is this breaking change what you would expect? If so, please fill in the following table, so that we can compose a migration guide for our customer:

|cmdlet| breaking change|old usage|new usage|
|-|-|-|-|

Reply:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/v-kotred/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/v-kotred/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
.xl63
	{text-align:left;
	vertical-align:top;
	border:.5pt solid windowtext;}
.xl64
	{border:.5pt solid windowtext;
	white-space:normal;}
.xl65
	{text-align:left;
	vertical-align:top;
	border:.5pt solid windowtext;
	white-space:normal;}
.xl66
	{color:#24292F;
	font-size:12.0pt;
	font-weight:600;
	font-family:"Segoe UI", sans-serif;
	mso-font-charset:0;
	text-align:center;
	vertical-align:middle;
	border:.5pt solid windowtext;
	background:#D0CECE;
	mso-pattern:black none;
	white-space:normal;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">



cmdlet | breaking change | old usage | new usage
-- | -- | -- | --
Edit-AzWebAppBackupConfiguration | The   generic type for 'property UserAssignedIdentities' has been changed from   'System.Collections.Generic.IDictionary`2[System.String,Microsoft.Azure.Management.WebSites.Models.ManagedServiceIdentityUserAssignedIdentitiesValue]'   to 'System.Collections.Generic.IDictionary`2[System.String,Microsoft.Azure.Management.WebSites.Models.UserAssignedIdentity]'. | Edit-AzWebAppBackupConfiguration -FrequencyInterval <Int32>   -FrequencyUnit <String> -KeepAtLeastOneBackup -Name IpRule   -ResourceGroupName MyResourceGroup -RetentionPeriodInDays <Int32>   -StartTime 2016-11-30T22:00:00Z -StorageAccountUrl   'https://storageaccount.file.core.windows.net' | No change in customer   usage. Same as old.



</body>

</html>

Reply:
Added